main.go

```diff
@@ -1,89 +1,51 @@
 package main
 
 import (
-	"bytes"
-	"fmt"
+	"io"
 	"log"
 	"os"
-	"path/filepath"
-	"strings"
-	"time"
 
-	"github.com/afdesk/trivy-go-plugin/pkg/common"
-)
+	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
 
-var (
-	tempJsonFileName = "scan2html-report-temp.json"
-	version          = "dev"
-	availableFlags   = []string{"--html-result", "--load-result"}
+	"github.com/afdesk/scan2html/render"
 )
 
 func main() {
-	if common.IsHelp() || len(os.Args) <= 1 {
-		helpMessage()
-	}
-	pluginFlags, trivyCommand := common.RetrievePluginArguments(availableFlags)
-	jsonResultFile := pluginFlags["--load-result"]
-	if jsonResultFile == "" {
-		jsonResultFile = filepath.Join(os.TempDir(), tempJsonFileName)
-		err := common.MakeTrivyJsonReport(trivyCommand, jsonResultFile)
-		if err != nil {
-			log.Fatalf("failed to build report: %v", err)
-		}
-		defer func(file string) {
-			if err := os.Remove(file); err != nil {
-				log.Printf("failed to remove file %v", err)
-			}
-		}(jsonResultFile)
-	}
-
-	firstHTML, err := common.ReadPluginFile("first.html")
+	rootCmd, err := initFlags()
```

Comment:
it seems `initFlags` isn't relevant, something like `initRootCmd` looks better, isn't it?